package.json

```diff
@@ -13,78 +13,60 @@
-    "@bcrs-shared-components/web-chat": "2.0.0",
-    "@mdi/font": "^5.5.55",
     "@sentry/browser": "^5.21.1",
     "@sentry/integrations": "^5.21.1",
+    "@vue/compat": "^3.2.39",
     "array.prototype.move": "0.0.4",
     "axios": "^0.21.1",
     "core-js": "^3.6.5",
     "http-status-codes": "^2.2.0",
-    "launchdarkly-js-client-sdk": "^2.18.0",
+    "launchdarkly-js-client-sdk": "^2.22.1",
     "lodash": "^4.17.20",
-    "pdfjs-dist": "2.0.943",
     "regenerator-runtime": "^0.13.7",
-    "sbc-common-components": "3.0.1",
-    "uuid": "^8.3.1",
-    "vue": "2.6.12",
+    "uuid": "^9.0.0",
+    "vue": "^3.2.39",
     "vue-affix": "^0.5.2",
     "vue-observe-visibility": "^1.0.0",
-    "vue-router": "^3.4.3",
+    "vue-router": "^4.1.5",
     "vue-the-mask": "^0.11.1",
     "vue2-filters": "^0.10.0",
     "vuelidate": "^0.7.5",
-    "vuetify": "^2.3.9",
-    "vuex": "^3.5.1"
+    "vuetify": "3.0.0-beta.4",
+    "vuex": "^4.0.2"
   },
   "devDependencies": {
+    "@mdi/font": "^7.0.96",
     "@types/jest": "^25.2.3",
     "@types/lodash": "^4.14.159",
     "@types/vue-the-mask": "^0.11.0",
     "@types/vuelidate": "^0.7.13",
-    "@typescript-eslint/eslint-plugin": "^2.34.0",
-    "@typescript-eslint/parser": "^2.34.0",
-    "@vue/cli-plugin-babel": "^4.5.4",
-    "@vue/cli-plugin-eslint": "^4.5.4",
-    "@vue/cli-plugin-typescript": "^4.5.4",
-    "@vue/cli-plugin-unit-jest": "^4.5.6",
-    "@vue/cli-service": "^4.5.4",
-    "@vue/eslint-config-standard": "^4.0.0",
-    "@vue/eslint-config-typescript": "^4.0.0",
-    "@vue/test-utils": "1.0.0-beta.33",
-    "eslint": "^6.8.0",
-    "eslint-plugin-vue": "^6.2.2",
+    "@typescript-eslint/eslint-plugin": "^5.37.0",
+    "@typescript-eslint/parser": "^5.37.0",
+    "@vue/cli-plugin-babel": "^5.0.8",
+    "@vue/cli-plugin-eslint": "^5.0.8",
+    "@vue/cli-plugin-typescript": "^5.0.8",
+    "@vue/cli-plugin-unit-jest": "^5.0.8",
+    "@vue/cli-service": "^5.0.8",
+    "@vue/compiler-sfc": "^3.2.39",
+    "@vue/eslint-config-standard": "^8.0.1",
+    "@vue/eslint-config-typescript": "^11.0.1",
+    "@vue/test-utils": "^2.0.2",
+    "eslint": "^8.23.1",
+    "eslint-plugin-vue": "^9.4.0",
     "flush-promises": "^1.0.2",
     "intersection-observer": "^0.12.0",
+    "s-vuex-class": "^0.4.1",
     "sass": "^1.26.10",
     "sass-loader": "^8.0.2",
     "sinon": "^9.0.3",
-    "typescript": "^3.9.7",
-    "vue-cli-plugin-vuetify": "^2.0.7",
+    "typescript": "^4.8.3",
+    "vue-class-component": "8.0.0-rc.1",
+    "vue-cli-plugin-vuetify": "^2.5.5",
     "vue-plugin-helper-decorator": "^0.0.12",
-    "vue-property-decorator": "^8.5.1",
-    "vue-template-compiler": "2.6.12",
+    "vue-property-decorator": "10.0.0-rc.3",
     "vue-test-utils-helpers": "git+https://github.com/bcgov/vue-test-utils-helpers.git",
-    "vuetify-loader": "^1.6.0",
-    "vuex-class": "^0.3.2",
-    "vuex-module-decorators": "^0.16.1"
+    "vuex-composition-helpers": "^1.1.2",
+    "vuex-module-decorators": "^2.0.0",
```

Comment:
There's a lot of discussion on whether the store decorators work in Vue3. (I think they do but I didn't get far enough to test them.)

Since Vuex is deprecated in Vue3, there's also discussion about using [**pinia**](https://vuejs.org/guide/scaling-up/state-management.html#pinia) instead.